app/services/platform-app-store/index.ts

```diff
@@ -0,0 +1,30 @@
+import { StatefulService } from 'services/stateful-service';
+import { mutation } from 'services/stateful-service';
+
+interface IPlatformAppStoreServiceState {
+  paypalAuthCallback: Function;
+}
+
+export class PlatformAppStoreService extends
+  StatefulService<IPlatformAppStoreServiceState> {
```

Comment:
You don't want to use a stateful service here.  Stateful services are only useful if you need the UI to react to changes in the store.  Furthermore, they have lots of drawbacks including boilerplate, and the fact that they are synced between our windows which means anything you put into a store needs to be serializable (functions are not).  You should use a normal service instead, and just store it as an attribute on the class instance.